src/core/localization/translations/en/common.json

```diff
@@ -132,7 +132,7 @@
       "transformations": {
         "no_transformation": "No transformation",
         "square_root": "Square root: sign(x)√(|x|)",
-        "cube_root": "Cube root: : ∛x",
+        "cube_root": "Cube root: ∛x",
         "log_one": "Log(x - xmin + 1)",
```

Comment:
```suggestion
        "log_one": "log₁₀(x - xₘᵢₙ + 1)",
```

Reply:
With our current font subscript i looks pretty wonky :) Do we want to still use it like this?
<img width="286" alt="image" src="https://github.com/user-attachments/assets/47682dd7-1af1-428b-a404-0309b7c96208">

It depeneds on font, e.g. in Arial it looks better (but don't know if Arial fits well in general)
<img width="276" alt="image" src="https://github.com/user-attachments/assets/c83006f8-6446-42bb-8163-9c04723c8b37">

Reply:
Looks bad in dropdown too. I suggest we keep the `Log₁₀` part, but do `xmin` or `Xmin` with normal text

<img width="180" alt="image" src="https://github.com/user-attachments/assets/f9c8f366-d8e3-4b0b-a089-3c02f3c18fef">

Reply:
fixed (partially)